.github/workflows/ci.yml

```diff
@@ -7,9 +7,11 @@ on:
   push:
     branches:
       - main
+  schedule:
+      - cron: "0 0 * * *"
```

Comment:
Since the track can update independently of the test runner, I also have this run once daily.